woocommerce/woocommerce
# Core E2E test package

### All Submissions:


### Changes proposed in this Pull Request:

This PR moves the core E2E test scripts to the `@woocommerce/e2e-core-tests` package in `tests/e2e/core-tests`. It adds scripts to the `tests/e2e/specs` folder for executing the scripts.

Closes #25962 .

### How to test the changes in this Pull Request:

1. Run 
```
npm install
npm docker:up
npm run test:e2e
npm run docker:down
```
2. Tests should run normally
3. Run individual tests
```
npm run docker:up
npm run test:e2e tests/e2e/specs/activate-and-setup/test-activation.js
npm run test:e2e tests/e2e/specs/activate-and-setup/test-onboarding-tasklist.js
npm run test:e2e tests/e2e/specs/activate-and-setup/test-initial-settings.js
npm run test:e2e tests/e2e/specs/wp-admin/test-add-variable-product.js
npm run test:e2e tests/e2e/specs/wp-admin/test-add-simple-product.js
npm run test:e2e tests/e2e/specs/wp-admin/test-update-tax-settings.js
npm run test:e2e tests/e2e/specs/wp-admin/test-create-coupon.js
npm run test:e2e tests/e2e/specs/wp-admin/test-update-product-settings.js
npm run test:e2e tests/e2e/specs/wp-admin/test-create-order.js
npm run test:e2e tests/e2e/specs/wp-admin/test-update-general-settings.js
npm run test:e2e tests/e2e/specs/front-end/test-cart.js
npm run test:e2e tests/e2e/specs/front-end/test-checkout.js
npm run test:e2e tests/e2e/specs/front-end/test-my-account.js
npm run test:e2e tests/e2e/specs/front-end/test-single-product-page.js
```

### Other information:




### Changelog entry

> Introduce core E2E tests package
